src/perform_step/gpu_tsit5_perform_step.jl

```diff
@@ -16,7 +16,8 @@
     adv_integ = true
     ## Check if tstops are within the range of time-series
     if integ.tstops !== nothing && integ.tstops_idx <= length(integ.tstops) &&
-       integ.tstops[integ.tstops_idx] <= integ.t + integ.dt
+       (integ.tstops[integ.tstops_idx] < integ.t + integ.dt ||
+        integ.tstops[integ.tstops_idx] ≈ integ.t + integ.dt)
```

Comment:
This change was done to account for floating point math error, which causes regression with `Tsit5()`

Reply:
`integ.tstops[integ.tstops_idx] - integ.t - integ.dt + 100*eps(integ.t) < 0` ?